packages/igx-templates/igx-ts-legacy/projects/_base/index.ts

```diff
@@ -65,7 +65,8 @@ $app-palette: palette($primary, $secondary, $surface);
 			dot: ".",
 			path: name,
 			projectTemplate: this.id,
-			yamlDefaultBranch: "<%=yaml-default-branch%>" // the placeholder will be evaluated by CodeGen
+			yamlDefaultBranch: "<%=yaml-default-branch%>", // the placeholder will be evaluated by CodeGen
+			ApplicationTitle: "<%=ApplicationTitle%>" // the placeholder will be evaluated by CodeGen
```

Comment:
since this is used only in base template, I dont see a reason to have this in the config. It basically evaluates the <%=ApplicationTitle%> placeholder as <%=ApplicationTitle%>, also please use camelCase. This applies for igx-ts as well

Reply:
As we discussed, currently there is no concreate naming convention for placeholders for the project. There are instances of PascalCase, camelCase, kebab-case and snake_case. 

And since in the mentioned code, where we put  [ApplicationTitle: "<%=ApplicationTitle%>"], there are already props with capital letters (e.g CustomTheme, DefaultTheme), we've decided to leave it like this and not change it, because this will force changes in several other places with no benefit.